app/forms/error_messages.py

```diff
@@ -13,11 +13,11 @@
     "MANDATORY_NUMBER": lazy_gettext("Enter an answer"),
     "MANDATORY_TEXTAREA": lazy_gettext("Enter an answer"),
     "MANDATORY_RADIO": lazy_gettext(
-        'Select an answer <span class="u-vh">to ‘%(question_title)s’</span>'
+        'Select an answer <span class="ons-u-vh">to ‘%(question_title)s’</span>'
     ),
```

Comment:
It is worth updating these classes in the `messages.po` file, so the existing translations work.